packages/ocaml-sat-solvers/ocaml-sat-solvers.0.7/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+name: "ocaml-sat-solvers"
+version: "0.7"
+maintainer: "opam-devel@lists.ocaml.org"
+authors: [ "Oliver Friedmann"
+           "Martin Lange" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/tcsprojects/ocaml-sat-solvers"
+dev-repo: "git://github.com/tcsprojects/ocaml-sat-solvers.git"
+bug-reports: "https://github.com/tcsprojects/ocaml-sat-solvers/issues"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"] {with-test}
+  ["ocaml" "setup.ml" "-build"] {with-test}
+  ["ocaml" "setup.ml" "-test"] {with-test}
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
```

Comment:
This is not necessary since opam 2

Reply:
Sorry, I am not entirely sure what exactly this is referring to. I also failed to find a documentation for what exactly the Opam 2.0 format is. If you could give me a pointer, I'll check it out. Thanks!

Reply:
This whole section is no longer needed since opam 2.0:
```diff
- remove: [
-  ["ocamlfind" "remove" "ocaml-sat-solvers"]
- ]
```

Now opam keeps track of files added during installation and automatically removes what is needed

Reply:
Thanks! I have deleted it.